src/components/Drawer.js

```diff
@@ -131,31 +126,31 @@ const Drawer = React.createClass({
           type='base'
         />
       </nav>
-    ) : <div style={styles.nav} />;
+    ) : <div className={css(styles.nav)} />;
   },
 
   render () {
     const styles = this.styles();
```

Comment:
We could also probably put `styles` on state and update it on incoming prop/state change if we so desired.  I can't think of a reason that wouldn't work.